x/apps/keeper/keeper.go

```diff
@@ -35,7 +36,8 @@ func NewKeeper(cdc *codec.Codec, key sdk.StoreKey, posKeeper types.PosKeeper, su
 
 	// ensure staked module accounts are set
 	if addr := supplyKeeper.GetModuleAddress(types.StakedPoolName); addr == nil {
-		panic(fmt.Sprintf("%s module account has not been set", types.StakedPoolName))
+		fmt.Println(fmt.Errorf("%s module account has not been set", types.StakedPoolName))
+		os.Exit(1)
```

Comment:
```suggestion
  log.Fatalf("%s module account has not been set", types.StakedPoolName)
```
stdlib package `log` offers `log.Fatalf` which logs the error with a timestamp & calls onto os.Exit(1)